### PR TITLE
Fix apple pay button when DW disabled

### LINKED ISSUE #9: Apple Pay shows in Safari when Digital Wallet is not enabled
_Master version of Square and master version of Event Espresso_

I finished up the drafts of the Square product page and documentation page. I stepped away and came back and realized an image of the payment option fields would be helpful.

While getting the screenshot, I noticed that Apple Pay appears in Safari even though the Digital Wallet is not enabled.

**Steps to reproduce**

- Add master Event Espresso to a WP site.
- Add master Square to a WP site.
- Enable the Square payment method.
- Connect Square.
- Start a registration for a paid event in Safari. During the payment, you'll see the Apple Pay button appear:

Registration Checkout – EE Playground Site 2021-11-08 at 10 01 16 AM

I've confirmed on two other websites when using Safari and I also see this on my iPhone. I noticed that Apple Pay is immediately shown while the card area loads for a second or two.

**Reported or seen on**

https://enzo12staging.wpengine.com/wp-admin/admin.php?page=espresso_payment_settings --> https://enzo12staging.wpengine.com/events/

https://playground.eventespresso.com/wp-admin/admin.php?page=espresso_payment_settings --> https://playground.eventespresso.com/events/

**Other notes**

### PR DESCRIPTION
**Pull request by knazart**
Resolves #9 

This fixes the digital wallet, when the Apple Pay button was shown in Safari while the DW was disabled in the settings.

### CODE REVIEW

File: assets/js/square-payments.js [hunk 1]
```diff
@@ -313,7 +315,7 @@ jQuery(document).ready(function($) {
 
 		/**
 		 * @function initializeGooglePay
-		 * Initializes the Apple Pay object for Square payments.
+		 * Initializes the Google Pay object for Square payments.
```
Comment (tn3rb) [inline assets/js/square-payments.js:318]: he he he 😄

**Review by tn3rb — APPROVED** [review]
YUP 👍🏻

**Comment by knazart** [comment]
Would you like to test this @lorenzocaum ?)

**Comment by lorenzocaum** [comment]
Hey Nazar, I tested in Safari on my computer. Then checked on my Phone.

This issue is fixed as I don't see the Apple Pay button when digital wallet is disabled.

I also confirmed that Apple Pay and Google Pay shows when digital wallet is enabled.

Please merge!

**Pull request merged**